Feature branch 1

nikoescobal opened the pull request:
+Given space between sidebar and embedded video.

nikoescobal committed:
Added index.html and named the project Youtube Project

nikoescobal committed:
Added nav-bar section with CSS preset and changes.

mertcihankurel committed:
Aside bar and embeded video part added

mertcihankurel committed:
aside width changed

nikoescobal committed:
Rounded teh avatar images in the comment section and added some formatting changes to css.

mertcihankurel committed:
README file uploaded

mertcihankurel committed:
Github links added to README file

nikoescobal committed:
Edited navbar to not be transparent and adjusted the subscribe buton.

nikoescobal committed:
Reworked styling, improved buttons, and added finishing CSS touches.

nikoescobal committed:
Update README.md

Updated screenshot of project

nikoescobal committed:
Update README.md

nikoescobal committed:
Update README.md

nikoescobal committed:
Update README.md

nikoescobal committed:
Update README.md

nikoescobal committed:
Update README.md

elmejdki commented:
## Pull Request under review by _Zakariae El Mejdki_

elmejdki commented on README.md:
```diff
@@ -1,2 +1,33 @@
+Github: [@mcihadkurel](https://github.com/mcihadkurel)
+____________________________________________________
+#### Nikolas Escobal
+Github: [@nikoescobal](https://github.com/nikoescobal/Youtubeclone-muhammed-niko/commits?author=nikoescobal)
```
**[OPTIONA]** add your LinkedIn and Twitter profile links

elmejdki commented on index.html:
```diff
@@ -0,0 +1,660 @@
+          style="width: 80px;"
+          height:80px;
+          alt=""
+        />
```
Please surround your images with a link tag.

elmejdki commented on index.html:
```diff
@@ -0,0 +1,660 @@
+  <body>
+    <nav class="flex">
+      <div id="left-navbar" class="flex">
+        <i class="fa fa-bars" aria-hidden="true"></i>
```
please change this into an imbeded image, like the one below it.

elmejdki commented on index.html:
```diff
@@ -0,0 +1,660 @@
+        <i class="fa fa-video-camera" aria-hidden="true"></i>
+        <i class="fa fa-th fa-lg" aria-hidden="true"></i>
+        <i class="fa fa-bell" aria-hidden="true"></i>
+        <i class="fa fa-envelope" aria-hidden="true"></i>
```
these icons also change them into embedded images.
![image](https://user-images.githubusercontent.com/34653764/84329916-e8031800-ab7d-11ea-910f-8010673dbe55.png)

elmejdki commented on index.html:
```diff
@@ -0,0 +1,660 @@
+        </aside>
+      </div>
+    </div>
+    <div class="footer"></div>
```
please use semantic HTML, change this line into footer tag and also add the main tag to your code.

elmejdki commented on style.css:
```diff
@@ -0,0 +1,235 @@
+  font-family: Roboto, Arial, “Sans Serif”, Noto, “YT Sans”, “Roboto Mono”;
+}
+
+/* .video { */
+/* margin-left: initial; */
+/* padding-left: 100px; */
+/* padding-top: 30px; */
+/* } */
```
Please remove all the comments from your CSS files and HTML.

elmejdki commented on index.html:
```diff
@@ -0,0 +1,660 @@
+        <div class="container">
+          <div class="main-video">
+            <iframe
+              width="853"
+              height="480"
+              src="https://www.youtube.com/embed/YbYWhdLO43Q"
+              frameborder="0"
+              allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
```
Please remove all inline styling and use your CSS instead.

nikoescobal replied:
Hi Zakariae! According to this article entitled "[HTML Style Guide and Coding Conventions,](https://www.w3schools.com/html/html5_syntax.asp)" it is recommended to always specify alt, width, and height for Images. As much as possible, we did not use inline styling, except for the case of images. We would appreciate your consideration on this matter.

elmejdki commented on style.css:
```diff
@@ -0,0 +1,235 @@
+  align-self: center;
+}
+
+#autoplay {
+  display: inline;
+}
+
+.inline {
+  display: inline;
+}
```
this is duplicate you can use only the class if you want please remove the id selector

elmejdki commented on style.css:
```diff
@@ -0,0 +1,235 @@
+  margin-bottom: 4px;
+}
+.aside-description {
+  font-family: Roboto;
+  line-height: 1.2em;
```
you are duplicating these line of codes too much of times try to give it it's own selector and uses it on your HTML instead

elmejdki commented on style.css:
```diff
@@ -0,0 +1,235 @@
+.flex-column {
+  flex-direction: column;
+}
+.user-comments {
+  display: grid;
+}
```
this one also is used below in grid-container selector.

elmejdki reviewed (changes requested):
# Greate job so far guys :clap:, but your project needs some small changes in order to get approved :tophat: 
### status: require changes

- Please add live demo link to your repo you can use [githack](https://raw.githack.com/)
- Please add a .gitignore file into the root of your repo and add ".DS_Store" inside it, so that file will be ignored by Github.
![image](https://user-images.githubusercontent.com/34653764/84330272-de2de480-ab7e-11ea-8252-13c84f2f50ba.png)

![image](https://user-images.githubusercontent.com/34653764/84330249-cce4d800-ab7e-11ea-9559-2fefd778b497.png)

- Please align these two divs in front of each other you can do it easily with flexbox, I mean (**up next** and **autoplay**):
![image](https://user-images.githubusercontent.com/34653764/84330334-0f0e1980-ab7f-11ea-8c11-0095e6bd42d1.png)

- Please remove CSS duplicated code.

### please fix all the required changes in here and below, before requesting another code review.
if you have any questions or you need help with something don't hesitate to contact me on slack, you will find me as _Zakariae El Mejdki_

mertcihankurel committed:
comments deleted

nikoescobal committed:
Corrected footer tag, added main tag, deleted comments, duplicate selectors in CSS, and made minor adjustment to Autoplay button and up next.

nikoescobal committed:
Deleted comments, duplicates in CSS, and made minor changes to autoplay.

mertcihankurel committed:
gitignore

mertcihankurel committed:
Create .gitignore

mertcihankurel committed:
Delete .DS_Store

mertcihankurel committed:
Delete gitignore

mertcihankurel committed:
İcons changed with a href link tags, added gitignore, live server added

mertcihankurel committed:
Merge branches 'feature-branch-1' and 'feature-branch-1' of https://github.com/nikoescobal/Youtubeclone-muhammed-niko into feature-branch-1

nikoescobal committed:
Aligned right nav bar images and added some minor css stylistic edits to the lower video section.

nikoescobal committed:
Update README.md

Added updated screenshot of the project, as well as made minor adjustments to profile handles.

nikoescobal committed:
Update README.md

nikoescobal committed:
Update README.md

BrittanyBlake commented:
### Pull request under review by Brittany Blake (TSE)

BrittanyBlake commented on index.html:
```diff
@@ -0,0 +1,670 @@
+              <div class="user-description">
+                <div class="page-name"></div>
+                <div class="align-center">
+                  <img
+                    class="images inline"
+                    src="assets/user-profile.jpg"
+                    alt="Page Profile"
+                    width="40"
+                    height="40"
+                  />
```
please remove inline styling. The image tag should only contain the alt element as it is used for screen readers.

BrittanyBlake commented on index.html:
```diff
@@ -0,0 +1,670 @@
+            <div class="total-comments">
+              <div class="user-comments">
+                <div class="user-avatar">
+                  <img
+                    class="images"
+                    src="assets/user-avatar-1.jpg"
+                    alt="User Profile"
+                    width="40"
+                    height="40"
+                  />
```
Please remove inline styling.

BrittanyBlake commented on index.html:
```diff
@@ -0,0 +1,670 @@
+              </div>
+              <div class="user-comments">
+                <div class="user-avatar">
+                  <img
+                    class="images"
+                    src="assets/user-avatar-2.jpg"
+                    alt="User Profile"
+                    width="40"
+                    height="40"
+                  />
```
Please remove inline styling.

BrittanyBlake reviewed (changes requested):
### Status: Changes Required ♻️ 
****
### Hi @nikoescobal  and @mcihadkurel! 👋 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 You have also done a wonderful job with your project so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [x] The [HTML validator](https://validator.w3.org) is showing some errors on your page. Please fix these errors and resubmit.
![Screen Shot 2020-06-11 at 6 14 38 PM](https://user-images.githubusercontent.com/48804258/84414216-f733a580-ac11-11ea-9c94-74ee43f3c863.png)

- [x] As was requested by the previous TSE, please remove inline styling from your HTML code and use your CSS file instead. I have highlighted some areas where you are using inline styling in my comments. Please use this is a guide and remove all other instances of inline styling. 

- [x] The icon of the user who uploaded the video and the users in the comments are not aligned properly. Please compare your clone with the original page and fix this.
Your clone 👇 
![Screen Shot 2020-06-11 at 6 29 38 PM](https://user-images.githubusercontent.com/48804258/84415051-7fb24600-ac12-11ea-8d5f-2b6e238431b4.png)
The original page 👇 
![Screen Shot 2020-06-11 at 6 29 42 PM](https://user-images.githubusercontent.com/48804258/84415057-817c0980-ac12-11ea-8d56-4648cfd6b753.png)
Your clone 👇 
![Screen Shot 2020-06-11 at 6 29 57 PM](https://user-images.githubusercontent.com/48804258/84415072-8640bd80-ac12-11ea-8cbb-0e6b1cd42ac9.png)
The original page 👇 
![Screen Shot 2020-06-11 at 6 30 10 PM](https://user-images.githubusercontent.com/48804258/84415076-8771ea80-ac12-11ea-9c7a-600bbb0e5241.png)
****
### [OPTIONAL]

- [x] In your future projects, please add a more descriptive pull request that includes an appropriate title and a short summary of what you have done. 🗒️ 




****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

mertcihankurel committed:
New feedback

mertcihankurel committed:
Merge branch 'feature-branch-1' of https://github.com/nikoescobal/Youtubeclone-muhammed-niko into feature-branch-1

mertcihankurel committed:
İnline styles moved to CSS file

mertcihankurel committed:
Syntax errors fixed

nikoescobal committed:
Added sort button, adjusted CSS for video below section, comments section, and search bar.

nikoescobal commented:
Hi @BrittanyBlake and @elmejdki! Thank you for taking time to review this project that @mcihadkurel and I spent time working on. We've applied all the suggested changes -- removed in-line styling, adjusted the CSS to match as closely as possible to match the original YouTube page, and have made sure that there are no errors on the validator.

nikoescobal committed:
Update README.md

nikoescobal committed:
Update README.md

Added updated screenshot, fixed a minor typo, and qualified the "Built with" section further.

nikoescobal committed:
Create .stickler.yml

nikoescobal committed:
Update .stickler.yml

nikoescobal committed:
Update .stickler.yml

nikoescobal committed:
Create stylelint.config.js

peterrobert commented on style.css:
```diff
@@ -0,0 +1,287 @@
+  float: left;
+  height: 10px;
+  position: relative;
+  width: 168;
+  height: 94;
```
Remember width and height are units. This being the case you need to add a unit after the number. Example use px for pixels.

peterrobert commented on index.html:
```diff
@@ -0,0 +1,469 @@
+                <i class="fa fa-thumbs-down icon-spacing" aria-hidden="true">
+                  5.8K</i
+                >
+                <i class="fa fa-share icon-spacing"> SHARE</i>
```
You need to make this a button. It is a requirement. You can wrap it using`<button>` tag

peterrobert commented on index.html:
```diff
@@ -0,0 +1,469 @@
+        <a href="#"><img src="assets/nav3.png" alt="" /></a>
+      </div>
+    </nav>
+
```
You need to wrap the main part of your webpage inside the main tag. You can check the usage of the semantic tag [here](https://www.w3schools.com/html/html5_semantic_elements.asp)

peterrobert reviewed (changes requested):
### CHANGES REQUIRED :recycle:  :stop_sign: 
Great work so far :+1:  Very good work implementing the changes required by the previous TSE. However, there are a few more issues that need fixing before I can approve this project :smiley: 

- [ ] Kindly add a margin between these two areas in your webpage :arrow_down: 
![space](https://user-images.githubusercontent.com/32361963/84637195-170ee600-aefe-11ea-9694-73cc199009f3.jpg)

There are some more required changes below :arrow_down: 

Kindly work on the suggested areas and resubmit your work for another review

### [OPTIONAL ] :arrow_down: 
I can see you added stickler files and enabled the linter in your project. If you can kindly fix the linter errors. However, this is not a requirement for this project so feel free to leave it.

If you have any question concerning the review kindly contact me via slack on @peterrobert

### HAPPY CODING  :computer:

mertcihankurel committed:
Buttons below description added. Width given to the aside bar.

nikoescobal committed:
Added transparency to button selector, added main tag to make markup more semantic.

nikoescobal commented:
Resolved latest request issues -- adjusted spacing between the video and sidebar, and added the main tag.

addod19 commented:
## Pull Request under review by Daniel Larbi Addo (TSE)

addod19 reviewed (changes requested):
## Status: Require changes :recycle: :traffic_light: 

Hi **TEAM**, you did a great job fixing all but one of the comments made by the previous TSE :+1: Before we can proceed, we need to fix the comments below:
- ```Sidebar``` has overlapped into the main video content. Let us fix it
![margin](https://user-images.githubusercontent.com/46388113/84693973-ea77c000-af37-11ea-8643-74057d1bdf63.png)
- Let us fix the stickler errors. Let view the details of the error and fix them

Kindly implement these changes and submit another core review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

mertcihankurel committed:
Given space between sidebar and embedded video

christianotieno reviewed (changes requested):
Hello Team, One crucial thing we have left out before we wrap this up. Kindly ensure that all the linter check are passing. While you have made the changes, please submit for another code review.

nikoescobal committed:
Fixed linter errors.

nikoescobal committed:
Fixed remaining linter errors.

nikoescobal committed:
Fixed last errors.

nikoescobal commented:
Hi @christianotieno and @addod19 ! As @peterrobert mentioned, fixing the stickler errors is optional as it is not a project requirement. However, because I did enable it (it was my first time to install), I have decided to go ahead and fix all errors. @mcihadkurel and I are thankful to all the TSE's who took the time to review and help improve our code. Thank you.

tirthajyoti-ghosh reviewed (approved):
# Status: Approved ✔

## Impressive project team 👏🙌👌. Great job implementing the changes 🙌!

### It's a 👍 from. Go ahead and merge it.

### Onto the next project 🚀. Best of luck ✨.

Happy coding 👨‍💻
